app/views/signup.erb

```diff
@@ -0,0 +1,6 @@
+<form action="/signup" method="post">
+  <label for="username">User name</label> <input type="text" name="user[username]" id="username" /><br />
```

Comment:
Please format your html. `<label>` tag and `<input>` should not be on the same line.